<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HueSelector</class>
 <widget class="QDialog" name="HueSelector">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>137</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>137</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>137</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Channel Stain Selection</string>
  </property>
  <property name="toolTip">
   <string>Select channel's stain</string>
  </property>
  <property name="statusTip">
   <string>Select channel's stain</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="hueLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Selected value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="hueBox">
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Selected Stain Value</string>
      </property>
      <property name="statusTip">
       <string>Selected Stain Value</string>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>359</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HueSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HueSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
